.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 
 node_js:
-  - "5.5.0"
+  - "5.11.1"
```

Comment:
The version bump for node here, is to get travis to use a newer version of npm as it seems to handle installation better.